CHANGELOG.md

```diff
@@ -3,10 +3,15 @@
 [Full changelog](https://github.com/mozilla/glean/compare/v30.1.0...master)
 
 * General:
-  * The version of glean_parser has been upgraded to v1.21.0
+  * The version of `glean_parser` has been upgraded to v1.22.0
+    * A maximum of 10 `extra_keys` is now enforced for event metric types.
```

Comment:
Mh, are we now adding changes from the glean_parser here as well? Cool, I didn't know that

Reply:
I added it because I felt it might be relevant to Glean consumers and they are more likely to check here first.